lib/parity/backup.rb

```diff
@@ -53,10 +53,15 @@ def download_remote_backup
     def restore_from_local_temp_backup
       Kernel.system(
         "pg_restore tmp/latest.backup --verbose --clean --no-acl --no-owner "\
-          "-d #{development_db} #{additional_args}",
+          "--dbname #{development_db} --jobs `#{number_of_cores}` "\
+          "#{additional_args}",
       )
     end
 
+    def number_of_cores
+      "getconf _NPROCESSORS_ONLN"
+    end
```

Comment:
Right; something in the README saying that the number of concurrent jobs can be set manually via env variable. (Or perhaps parity itself could have a `-j` flag?) And maybe `NUMBER_OF_JOBS` is a more accurate description of what's going on here.

OpenBSD:

```
~$  sysctl -n hw.logicalcpu_max
sysctl: second level name logicalcpu_max in hw.logicalcpu_max is invalid
~$ sysctl -n hw.physicalcpu_max
sysctl: second level name physicalcpu_max in hw.physicalcpu_max is invalid
~$ sysctl -n hw.ncpu            
1
```